README.rst

```diff
@@ -429,10 +429,6 @@ Example result response (cut to one element only instead of 10):
 Filters
 -------
 
-**note**: this feature is compatible with django-filter up to version 0.15.
```

Comment:
The library (documentation, tests, changelog) needs to be updated anyway at each release. If django-filter becomes stable and won't introduce backward incompatible changes I wouldn't mind deleting the note, but this dependency has caused quite a few issues recently and I prefer to do it this way.